docs/standard/microservices-architecture/implement-resilient-applications/implement-custom-http-call-retries-exponential-backoff.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Implementing custom HTTP call retries with exponential backoff
-description: .NET Microservices Architecture for Containerized .NET Applications | Implementing custom HTTP call retries with exponential backoff
+title: Explore custom HTTP call retries with exponential backoff
+description: .NET Microservices Architecture for Containerized .NET Applications | Explore custom HTTP call retries with exponential backoff. The purpose of this secction is just to explain and explore the problem.
```

Comment:
typo: secction

The description could be something like "Learn how you could handle possible HTTP failure scenarios using your own implementation..."

It needs to help people pick your topic when seeing it on search engine results. For example, this is how it looks right now:
![image](https://user-images.githubusercontent.com/12971179/41215556-bffd16ca-6d05-11e8-9c86-b44f437cfdaf.png)

Reply:
👍